Realm/Tests/TestHost/main.m

```diff
@@ -8,7 +8,13 @@
 
 #import <TargetConditionals.h>
 
-#if TARGET_OS_IPHONE
+#if TARGET_OS_TV || TARGET_OS_WATCH
+
+// tvOS and watchOS don't support testing at this time.
```

Comment:
From what I can see there's still no ability to test watchOS apps. tvOS does appear to support testing, and we should be able to update TestHost to run our tests once we have support for tvOS in master. I can update this comment to clarify the situation.